analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.TryCatchFinally.cs

```diff
@@ -852,6 +852,35 @@ public void Catch_Simple_NoFilter()
             validator.TagStates("End").Should().HaveCount(1).And.ContainSingle(x => x.Exception == null);
         }
 
+        [TestMethod]
+        public void Catch_WrappedInLocalLifetimeRegion()
+        {
+            const string code = @"
+var tag = ""BeforeTry"";
+try
+{
+    Tag(""InTry"");
```

Comment:
Would be better to use `throw` instead. Helps with clarity.

Reply:
I would have to tag & throw on the next line.